build.sbt

```diff
@@ -1,12 +1,41 @@
-name         := "anghammarad"
-organization := "com.gu"
-scalaVersion := "2.12.4"
-version      := "0.1.0-SNAPSHOT"
+import com.gu.riffraff.artifact.RiffRaffArtifact.autoImport.{riffRaffArtifactResources, riffRaffUploadManifestBucket}
+
+organization in ThisBuild := "com.gu"
+scalaVersion in ThisBuild := "2.12.4"
+
+//Projects
+
+lazy val root = project
+  .in(file("."))
+  .aggregate(`client-lib`, anghammarad)
+
+lazy val `client-lib` = project
+  .settings(
+    name := "client-lib",
+    libraryDependencies ++= dependencies,
```

Comment:
This is fine for now, but I suspect the dependencies will differ.

(client lib won't need AWS features apart from SNS, shouldn't include logging (might conflict with other people's loggers) and won't need a markdown parser)